build.gradle.kts

```diff
@@ -22,6 +22,12 @@ import org.springframework.boot.gradle.tasks.run.BootRun
 // implementations/configurations in a 'buildSrc' included build.
 // See https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources
 
+buildscript {
```

Comment:
Maybe a quick summary of why this explicit dependency requirement is there as well as a link to the openapi underlying issue would be great.

My idea is to clearly indicate that this is a workaround and that in the future is could be ok to remove this and/or to be careful if/when one wants to update this dependency. Having the information in the commit is great but I find it to be very useful to also have the information locally directly in the code.